webapp/app/js/config/sample.local.conf.js

```diff
@@ -3,17 +3,19 @@
  * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
 
 'use strict';
+var production = "https://treeherder.mozilla.org";
+var stage = "https://treeherder.allizom.org";
+var dev = "http://treeherder-dev.allizom.org";
+var same_host = "";
```

Comment:
I was just curious about the rationale for `same_host` vs. `local` or `localhost` here, though I'm sure there is a compelling reason.

I had tried local and localhost as vars, pointing to stage (just as a test) and it worked fine also.